openrc/hddfancontrol

```diff
@@ -0,0 +1,18 @@
+#!/sbin/openrc-run
+
+description="Adjust the fan speed according to the temperature of the hard drive"
+
+cfgfile="/etc/conf.d/$RC_SVCNAME.conf"
+pidfile="/run/$RC_SVCNAME.pid"
+
+command=/usr/bin/hddfancontrol
+command_args="$(cat $cfgfile | egrep '^\s*HDDFANCONTROL_ARGS' | sed -E -e 's/^\s*HDDFANCONTROL_ARGS\s*=\s*"\s*//' -e 's/["#].*$//')"
```

Comment:
Can't we directly source the config file?
Something like:
```
source $cfgfile
command_args=${HDDFANCONTROL_ARGS}
```

Reply:
Interesting idea. I refactored startup script a little.